scripts/gen-api-gtkdoc.py

```diff
@@ -271,6 +271,28 @@ def from_memberdef(xml):
         return DoxyTypedef(name, d)
 
 
+class DoxyDefine(DoxyElement):
+    @staticmethod
+    def from_memberdef(xml):
+        name = xml.find("name").text
+        v = xml.find("initializer")
+        if (v is not None):
+            proc = DoxygenProcess()
+            text = proc.process_element(v)
+            params = ",".join(p.text for p in xml.xpath(".//param/defname"))
+            if (params != ""):
+                params = "(%s)" % params
+            d = "#define %s%s %s" % (name, params, text)
+        else:
+            d = "#define %s" % name
+
+        e = DoxyEnum(name, d)
```

Comment:
Shouldn't this be `DocyDefine(name, d)`? (not that it actually matters much, but still)

Reply:
I was the one with the idea of providing this in GI. Basically, it is provided in plugin documentation and I thought it would be consistent to provide it with geany gi. Also, I use it in a project and you are correct that I can easily include it as well as customize it.

https://github.com/sagarchalise/geany-pyjedi/blob/peasy/peasyjedi.py